$NetBSD: patch-ad,v 1.5 2012/07/30 02:53:57 dholland Exp $

- Add image libraries to link. (is this needed?)
- Honor LDFLAGS so rpaths work.
- Configure paths for pkgsrc.

--- fdesign/Imakefile.orig	2002-11-16 13:29:20.000000000 -0800
+++ fdesign/Imakefile	2002-12-25 10:40:31.000000000 -0800
@@ -71,7 +71,7 @@
 
 FORMSLIBDIR     = $(TOP)/lib
 FORMSINCDIR     = $(FORMSLIBDIR)       
-FORMSLIB        = -L$(FORMSLIBDIR) -lforms
+FORMSLIB        = -L$(FORMSLIBDIR) -lforms $(JPEGLIB) $(XPMLIB) $(LDFLAGS)
 DEPFORMSLIB     = $(FORMSLIBDIR)/libforms.a
 
 #if !HaveSnprintf
@@ -99,7 +99,7 @@
 		  -I.
 
 DEPEND_DEFINES	= \
-		  -I/usr/local/include
+		  -I${LOCALBASE}/include
 
 
 EXTRA_LIBRARIES	= \
